ps_checkout.php

```diff
@@ -1577,6 +1578,14 @@ private function installSentryExceptionLogger()
             );
 
             $this->sentryClient->setAppPath(realpath(_PS_MODULE_DIR_ . 'ps_checkout/'));
+            $this->sentryClient->setExcludedAppPaths([
+                realpath(_PS_MODULE_DIR_ . 'ps_checkout/vendor/'),
+            ]);
+            $this->sentryClient->setExcludedDomains(['127.0.0.1', 'localhost', '.local']);
+
+            if (version_compare(phpversion(), '7.4.0', '>=')) {
```

Comment:
It would be interesting that later versions of PrestaShop compatible with PHP 7.4 send errors to sentry again without having to come back on this code?

```suggestion
            if (version_compare(phpversion(), '7.4.0', '>=') && version_compare(_PS_VERSION_, '1.7.8.0', '<')) {
```